connector/src/main/java/org/geysermc/connector/network/BedrockProtocol.java

```diff
@@ -40,8 +41,8 @@
      * Default Bedrock codec that should act as a fallback. Should represent the latest available
      * release of the game that Geyser supports.
      */
-    public static final BedrockPacketCodec DEFAULT_BEDROCK_CODEC = Bedrock_v422.V422_CODEC.toBuilder()
-            .minecraftVersion("1.16.201")
+    public static final BedrockPacketCodec DEFAULT_BEDROCK_CODEC = Bedrock_v428.V428_CODEC.toBuilder()
+            .minecraftVersion("1.16.210")
```

Comment:
This can be removed since it is already correct in protocol.
https://github.com/CloudburstMC/Protocol/blob/develop/bedrock/bedrock-v428/src/main/java/com/nukkitx/protocol/bedrock/v428/Bedrock_v428.java#L29

Reply:
Good point. xd